src/analyz/mod.rs

```diff
@@ -621,6 +610,10 @@ struct SubstConstTys<'tcx> {
 }
 
 impl<'tcx> mir::visit::MutVisitor<'tcx> for SubstConstTys<'tcx> {
+    fn tcx(&self) -> TyCtxt<'tcx> {
+        self.tcx
+    }
+
```

Comment:
This function seems added and commented out. Is it going to be used eventually?

Reply:
This was part of an incomplete update of `subst_const_tys` for the new Rust nightly, but `subst_const_tys` can just be removed instead.